extra-games/qzdl/autobuild/defines

```diff
@@ -0,0 +1,7 @@
+PKGNAME=qzdl
+PKGSEC=games
+PKGDES="A popular launcher for GZDoom with user friendly interface"
```

Comment:
Note that your description is not exactly objective or descriptive (hint, "user friendly" is not exactly helpful).